packages/subgraph/manifest/data/localhost.json

```diff
@@ -5,18 +5,18 @@
     "DAORegistry": {
       "name": "DAORegistry",
       "address": "0x9A9f2CCfdE556A7E9Ff0848998Aa4a0CFD8863AE",
-      "startBlock": 18
+      "startBlock": 0
```

Comment:
is the addres here are dummy addresses ? if yes, can we use address 1,2,3...etc?